tensorboard/plugins/scalar/scalars_plugin.py

```diff
@@ -192,10 +192,14 @@ def tags_route(self, request):
   @wrappers.Request.application
   def scalars_route(self, request):
     """Given a tag and single run, return array of ScalarEvents."""
-    # TODO: return HTTP status code for malformed requests
```

Comment:
I don't think your change fully addresses this comment (which would also include requests missing args like `tag` and `run` so could you leave it there?  FWIW I'm not sure in what case scalars_impl() even throws ValueError so it may be best to leave this file unchanged.

Reply:
Reverted that